### PR TITLE
Removed redundant OpenGLMaps

### PR DESCRIPTION
**Pull request by emilwestergren**
Due to the changes in the GpuMap system most of the old OpenGLMap implementations can now be removed.
Fixes https://github.com/cogneco/ooc-kean/issues/794

* Breaks API

### CODE REVIEW

**Comment by emilwestergren** [comment]
@davidpiuva peer review plz?

**Comment by davidpiuva** [comment]
I can take a look, when I have rebased InMotion.

**Comment by davidpiuva** [comment]
I could not spot any obvious mistakes in the code.

File: ooc-opengl.use [hunk 1]
```diff
@@ -14,7 +14,6 @@ Imports: OpenGLUv
 Imports: OpenGLCanvas
 Imports: OpenGLSurface
 Imports: OpenGLMap
-Imports: OpenGLMapPack
```
Comment (marcusnaslund) [inline ooc-opengl.use]: Do you still want this file? It's not used anywhere.

Comment (marcusnaslund) [inline ooc-opengl.use]: Otherwise delete it too.

Comment (marcusnaslund) [inline ooc-opengl.use]: If you don't mind, do that in another PR. I'm happy otherwise and almost done testing.

Comment (emilwestergren) [inline ooc-opengl.use]: Strange, it should have been deleted here. I'd rather fix it immediately.

**Comment by emilwestergren** [comment]
Removed OpenGLMapPack.ooc and modified revision history.
